lib/matplotlib/__init__.py

```diff
@@ -273,9 +273,11 @@ def _get_executable_info(name):
 
     Returns
     -------
-    If the executable is found, a namedtuple with fields ``executable`` (`str`)
-    and ``version`` (`distutils.version.LooseVersion`, or ``None`` if the
-    version cannot be determined).
+
+    tuple or None
+        If the executable is found, a namedtuple with fields ``executable``
```

Comment:
```suggestion
    tuple
        A namedtuple with fields ``executable``
```
This always returns `_ExecInfo` (or raises).

Please also re-wrap.